tools/RAiDER/dem.py

```diff
@@ -57,7 +57,18 @@ def download_dem(
     buf=0.02,
     overwrite=False,
 ):
-    """  Download a DEM if one is not already present. """
+    """  
+    Download a DEM if one is not already present. 
+    Args:
+            llbounds: list/ndarry of floats   -lat/lon bounds of the area to download. Values should be ordered in the following way: [S, N, W, E]
+            writeDEM: boolean                 -write the DEM to file
+            outName: string                   -name of the DEM file   
```

Comment:
Please add that if the DEM exists the default behavior is to use the existing.

Reply:
Will do 👍. I should have some time tomorrow to work on these